app.py

```diff
@@ -52,11 +52,28 @@ def azul_to_obj(result):
     data_object['checksums'] = [
         {'checksum': azul['fileMd5sum'], 'type': 'md5'}]
     data_object['aliases'] = azul['aliases']
-    data_object['updated'] = azul['lastModified'] + 'Z'
+    data_object['updated'] = azul['lastModified'].rstrip('Z') + 'Z'
```

Comment:
Thanks, suggests to me https://github.com/DataBiosphere/dos-azul-lambda/issues/89